Kopernicus/Kopernicus/Kopernicus.csproj

```diff
@@ -32,6 +32,10 @@
       <HintPath>..\..\Assembly-CSharp.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Kopernicus.Parser, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Please revert the changes to this file. It will mess with everyones build system, since you haven't committed Kopernicus.Parser.dll (and I would prefer not to have it in the repository apart from the copy in Distribution).

With the submodule we only mess with people who do a flat checkout, and most client recognize submodules and pull them automatically theese days.